### PR TITLE
Fixed (temporary) duplicate content when use catchAll with debugger

### PR DESCRIPTION
**Pull request by muhammadcahya**
| Q | A |
| --- | --- |
| Is bugfix? | yes (temporary) |
| New feature? | no |
| Breaks BC? | no |
| Fixed issues | #11371 , [#42](https://github.com/yiisoft/yii2-debug/issues/42) |

### CODE REVIEW

File: docs/guide/runtime-routing.md [hunk 1]
```diff
@@ -124,6 +124,9 @@ With the above configuration, the `site/offline` action will be used to handle a
 The `catchAll` property should take an array whose first element specifies a route, and
 the rest of the elements (name-value pairs) specify the parameters to be [bound to the action](structure-controllers.md#action-parameters).
 
+> Tip: In practice this is used in production, not at stage of development.
```
Comment (SilverFire) [inline docs/guide/runtime-routing.md]: I don't like this wording. It does not describe explicitly why it is not recommended to use this property on dev.

How about `Info: Debug panel on development environment will not work when this property is enabled.`

File: docs/guide/structure-applications.md [hunk 2]
```diff
@@ -192,6 +192,8 @@ The rest of the array elements (key-value pairs) specify the parameters to be bo
 ]
 ```
 
+> Tip: In practice this is used in production, not at stage of development.
```
Comment (SilverFire) [inline docs/guide/structure-applications.md]: Same here

**Comment by muhammadcahya** [comment]
Sorry for wording, thanks for the correction :)

**Comment by muhammadcahya** [comment]
Has been changed to `Info: Debug panel on development environment will not work when this property is enabled` in last commits

**Comment by SilverFire** [comment]
Could you update your branch from upstream in order to resolve conflicts in `docs/guide-id/intro-upgrade-from-v1.md`? [See how to](https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md#7-pull-the-latest-yii-code-from-upstream-into-your-branch)

**Comment by muhammadcahya** [comment]
well, sorry I'm missing about conflict

**Pull request merged**